src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -671,6 +672,11 @@ protected void buildShapelessRecipes(Consumer<IFinishedRecipe> recipeSaver)
 		GristCostRecipeBuilder.of(MSBlocks.CHISELED_GREEN_STONE_BRICKS).grist(BUILD, 2).grist(CAULK, 1).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSBlocks.HORIZONTAL_GREEN_STONE_BRICKS).grist(BUILD, 2).grist(CAULK, 1).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSBlocks.VERTICAL_GREEN_STONE_BRICKS).grist(BUILD, 2).grist(CAULK, 1).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSBlocks.SANDSTONE_COLUMN).grist(BUILD, 4).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSBlocks.CHISELED_SANDSTONE_COLUMN).grist(BUILD, 4).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSBlocks.RED_SANDSTONE_COLUMN).grist(BUILD, 4).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSBlocks.CHISELED_RED_SANDSTONE_COLUMN).grist(BUILD, 4).build(recipeSaver);
+
 		GristCostRecipeBuilder.of(MSBlocks.END_GRASS).grist(BUILD, 4).grist(CAULK, 3).build(recipeSaver);
```

Comment:
Is the space added here an intentional change?

Reply:
It is not.

Reply:
All good, its incredibly minor but would be good practice to fix

Reply:
Fixed.